UI/GalaxySetupWnd.cpp

```diff
@@ -188,6 +188,9 @@ void GameRulesPanel::CompleteConstruction() {
     // for all rules, add to page
     for (auto rule : GetGameRules()) {
         // get or create page for rule
+        // TODO: enable page for Multiplayer rules when lobby will get game rules panel
```

Comment:
or just leave them always visible as @Dilvish-fo suggested...?

Reply:
Yes, left them in UI.